presto-product-tests/bin/run_on_docker.sh

```diff
@@ -167,6 +168,16 @@ if [[ "$ENVIRONMENT" == "multinode" ]]; then
    PRESTO_SERVICES="${PRESTO_SERVICES} presto-worker"
 fi
 
+CLI_ARGUMENTS="--server presto-master:8080"
```

Comment:
Bummer.  One more reason to re-write the whole set of bash scripts as a singe Python script with real command line option processing...  But clearly that's out of scope for this PR.